src/containers/gui.jsx

```diff
@@ -126,17 +103,16 @@ GUI.propTypes = {
     isLoading: PropTypes.bool,
     isScratchDesktop: PropTypes.bool,
     isShowingProject: PropTypes.bool,
+    isShowingWithoutId: PropTypes.bool,
```

Comment:
Is this added intentionally? It looks unrelated.

Reply:
Removing...

Reply:
Done